Test suite has 2 failures

```
$ npm test
...                                                                                                                                                                                
  8 passing (130ms)                                                                                                                                                               
  2 failing                                                                                                                                                                       
                                                                                                                                                                                  
  1) Feathers application tests 404 shows a 404 HTML page:                                                                                                                        
                                                                                                                                                                                  
      Uncaught AssertionError: false == true                                                                                                                                      
      + expected - actual                                                                                                                                                         
                                                                                                                                                                                  
      -false                                                                                                                                                                      
      +true                                                                                                                                                                       
                                                                                                                                                                                  
      at Request.request [as _callback] (test/app.test.js:28:16)                                                                                                                  
      at Request.self.callback (node_modules/request/request.js:186:22)                                                                                                           
      at Request.<anonymous> (node_modules/request/request.js:1060:10)                                                                                                            
      at IncomingMessage.<anonymous> (node_modules/request/request.js:980:12)                                                                                                     
      at endReadableNT (_stream_readable.js:974:12)                                                                                                                               
      at _combinedTickCallback (internal/process/next_tick.js:74:11)                                                                                                              
      at process._tickCallback (internal/process/next_tick.js:98:9)                                                                                                               
                                                                                                                                                                                  
  2) message process hook filters data as expected:                                                                                                                               
                                                                                                                                                                                  
      AssertionError: { text: 'foo&amp;', sentBy: '1', createdAt: 1477616262773 } deepEqual { text: 'foo&amp;', userId: '1', createdAt: 1477616262773 }                           
      + expected - actual                                                                                                                                                         
                                                                                                                                                                                  
       {                                                                                                                                                                          
         "createdAt": 1477616262773                                                                                                                                               
      -  "sentBy": "1"                                                                                                                                                            
         "text": "foo&amp;"                                                                                                                                                       
      +  "userId": "1"                                                                                                                                                            
       }                                                                                                                                                                          
                                                                                                                                                                                  
      at Context.it (test/services/message/hooks/process.test.js:29:12)                                                                                                           

```

I suppose you've changed formats and/or defaults since these were last run. Setting `json: false` in the options of 1) didn't work, I still got json back in the test context though I do get HTML in the browser. 2) may just require updating the test for the new format.

Thanks for your amazing work on Feathers I am a huge fan!

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,2 @@
+language: node_js
+node_js: 'node'

--- a/README.md
+++ b/README.md
@@ -2,6 +2,8 @@
 
 # feathers-chat
 
+[![Build Status](https://travis-ci.org/feathersjs/feathers-chat.png?branch=master)](https://travis-ci.org/feathersjs/feathers-chat)
+
 > A real-time chat application
 
 ## About

--- a/test/app.test.js
+++ b/test/app.test.js
@@ -22,10 +22,9 @@ describe('Feathers application tests', () => {
   });
 
   describe('404', () => {
-    it('shows a 404 HTML page', done => {
+    it('sends a 404 error', done => {
       request('http://localhost:3030/path/to/nowhere', (err, res, body) => {
         assert.equal(res.statusCode, 404);
-        assert.ok(body.indexOf('<html>') !== -1);
         done(err);
       });
     });

--- a/test/services/message/hooks/process.test.js
+++ b/test/services/message/hooks/process.test.js
@@ -23,12 +23,12 @@ describe('message process hook', () => {
         text: 'foo&'
       }
     };
-    
+
     process()(mockHook);
-    
+
     assert.deepEqual(mockHook.data, {
       text: 'foo&amp;',
-      userId: '1',
+      sentBy: '1',
       createdAt: new Date().getTime()
     });
   });

